client/src/components/LoginSession/tests/LoginSession-story.js

```diff
@@ -10,6 +10,14 @@ const props = {
     Created: '2021-01-20 00:33:41',
     LastAccessed: '2021-03-11 03:47:22',
     LogOutEndpoint: 'admin/loginsession/remove',
+    logout: (beforeRequest, receivedResponse, caughtException) => {
```

Comment:
A better way of passing mock event handler to your pattern libraries stories is to use `@storybook/addon-actions`.

```js
import { action } from '@storybook/addon-actions';
const logout = action('logout');
logout.toString = () => 'logout';
```

That will log an entry in the storybook action logger.
![image](https://user-images.githubusercontent.com/1168676/113068094-15f33f00-921a-11eb-83b5-e55ecb8d82f9.png)

Reply:
It's not a mock event handler.  The logout() function needs to call beforeRequest() and receivedResponse() in order for the LoginSession state and thus text to update

Reply:
No longer required, can use `logout: () => 1` just to satisfy the func.required propType